doc/API/Bills.md

```diff
@@ -132,6 +132,66 @@ Output:
 }
 ```
 
+### `get_bill_graph`
+
+Retrieve a graph associated with a bill
+
+Route: `/api/v0/bills/:id/graphs/:graph_type
+
+Example:
+
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits?from=1517443200
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/bits?from=1517443200&to=1517788800
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/bills/1/graphs/monthly
```

Comment:
Probably worth documenting this returns a png no matter whether you say you want svg graphs or not.